bundles/org.openhab.binding.iaqualink/src/main/resources/ESH-INF/thing/iAqualink.xml

```diff
@@ -0,0 +1,253 @@
+			</channel>
+			<channel id="spa_salinity" typeId="chemical">
+				<label>Spa Salinity</label>
+				<description>Spa Salinity</description>
+			</channel>
+			<channel id="pool_salinity" typeId="chemical">
+				<label>Pool Salinity</label>
+				<description>Pool Salinity</description>
+			</channel>
+			<channel id="orp" typeId="chemical">
+				<label>Orp</label>
+				<description>Orp</description>
+			</channel>
+			<channel id="ph" typeId="chemical">
+				<label>PH</label>
+				<description>PH</description>
+			</channel>
+		</channels>
+		<config-description>
+			<parameter name="userName" type="text" required="true">
+				<label>User Name</label>
+				<description>The user name to use when connecting to a iAqualink Account</description>
+			</parameter>
+			<parameter name="password" type="text" required="true">
+				<label>Password</label>
+				<description>The password to use when connecting to a iAqualink Account</description>
+			</parameter>
+			<parameter name="refresh" type="integer" required="true">
+				<label>Refresh Interval</label>
+				<description>Specifies the refresh interval in seconds</description>
+				<default>30</default>
+			</parameter>
+			<parameter name="serialId" type="text" required="false">
+				<label>Serial Number</label>
+				<description>Optionally specify the serial number of the controller which can be found on the iAquaLink Owner's
+					Center. This is only useful if you have more then one controller (pool) associated with your account. Leave blank
+					to have the first controller used.
+				</description>
+			</parameter>
+			<parameter name="apiKey" type="text" required="false">
+				<label>Serial Number</label>
+				<description>Optionally specify the API key used for access. This is only useful for debugging or if the API key is
+					changed by the vendor
+				</description>
+				<default>EOOEMOW4YR6QNB07</default>
+			</parameter>
+		</config-description>
+	</thing-type>
+
+	<!-- System Channels -->
+
+	<channel-type id="status">
+		<item-type>String</item-type>
+		<label>Status</label>
+		<description>
+			The status of iAqualink connection
+		</description>
+		<state readOnly="true" />
+
+	</channel-type>
+
+	<channel-type id="system_type" advanced="true">
+		<item-type>String</item-type>
+		<label>System Type</label>
+		<description>
+			System Type
+		</description>
+		<state readOnly="true" />
+
+	</channel-type>
+
+	<!-- Equipment and Aux Channels -->
+
+	<channel-type id="aux-switch">
+		<item-type>Switch</item-type>
+		<label>Auxiliary Switch</label>
+		<description>The current state of auxiliary channel</description>
+	</channel-type>
+
+	<!-- See http://www.autelis.com/wiki/index.php?title=Pool_Control_HTTP_Command_Reference&oldid=861 for colors -->
+	<channel-type id="aux-number">
+		<item-type>Number</item-type>
+		<label>Auxiliary Number</label>
+		<description>The current state of auxiliary channel</description>
+		<state>
+			<options>
+				<option value="0">None</option>
```

Comment:
@Hilbrand Let me ask your opinion,  For the aux-number above , the value is definitely a number,  but there is a clear mapping to a descriptive string, and the number does not make much sense without the description ( for the user).  Would you recommend dropping the state options (which is not valid for a number), or making this a String type (with the string being "0" or "1", which seems counter intuitive )

Reply:
NM, after more research i needed to change this anyways  as the protocol ended up being more complicated with aux channels